build/RepositoryBuild.targets

```
@@ -56,7 +57,8 @@
       StopOnFirstFailure="false"
       Targets="_TestRepository"
       Properties="BuildNumber=$(BuildNumber);IsFinalBuild=$(IsFinalBuild);Configuration=$(Configuration);_NoBuildRepos=$(_NoBuildRepos)"
-      ContinueOnError="true">
+      ContinueOnError="true"
+      Condition=" @(_BatchedTestRepo.SkipTests) != 'true' ">
```

@natemcmaster This doesn't seem to do anything

---

```suggestion
      Condition=" '%(_BatchedTestRepo.SkipTests)' != 'true' ">
```

Might be due to wrong syntax...

---

Oh, the quotes...